api/schema/application_azure_resources_resource.go

```diff
@@ -44,7 +54,7 @@ var (
 						},
 					},
 					attributeWithMode: attributeWithMode{
-						Mode: ImmutableAttributeMode,
+						Mode: IDAttributeMode,
```

Comment:
Shouldn't this be in the create request?
```suggestion
						Mode: KeyAttributeMode,
```